src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala

```diff
@@ -192,12 +132,96 @@ case class KhermesConsoleHelper(client: KhermesClientActor) {
+      new NullCompleter())
+    val argumentStopCompleter =new ArgumentCompleter(new StringsCompleter("stop"), new StringsCompleter("--ids"), new NullCompleter())
+    reader.addCompleter(argumentShowCompleter)
+    reader.addCompleter(argumentSaveCompleter)
+    reader.addCompleter(argumentStartCompleter)
+    reader.addCompleter(argumentStopCompleter)
+    reader.setPrompt("\u001B[33mkhermes> \u001B[0m")
     reader
   }
 }
 
 object KhermesConsoleHelper {
 
   val TimeoutWhenLsMessage = 200L
-}
+
+  def printNotFoundCommand: String = {
+    "Command not found. Type help to list available commands."
+  }
+
+  def help: String = {
+    s"""Khermes client provides the next commands to manage your Khermes cluster:
+       |  Usage: COMMAND [args...]
+       |
+               |  Commands:
+       |     start [command options] : Starts event generation in N nodes.
+       |       --khermes    : Khermes configuration
+       |       --kafka      : Kafka configuration
+       |       --template   : Template to generate data
+       |       --avro       : Avro configuration
+       |       --ids        : Node id where start khermes
+       |     stop [command options] : Stop event generation in N nodes.
+       |       --ids        : Node id where start khermes
+       |     ls                    : List the nodes with their current status
+       |     save [command options] : Save your configuration in zookeeper
+       |       --khermes    : Khermes configuration
+       |       --kafka      : Kafka configuration
+       |       --template   : Template to generate data
+       |       --avro       : Avro configuration
+       |     show [command options] : Show your configuration
+       |       --khermes    : Khermes configuration
+       |       --kafka      : Kafka configuration
+       |       --template   : Template to generate data
+       |       --avro       : Avro configuration
+       |     clear                 : Clean the screen.
+       |     help                  : Print this usage.
+       |     exit | quit | bye     : Exit of Khermes Cli.   """.stripMargin
+  }
+
+  def show(args: Map[String, String], firstWord: String): String = {
+    Try {
+      val a = args.map {
+        case ("khermes", _) => load(s"$GeneratorConfigPath/${args.head._2}").getOrElse(s"Khermes ${args.head._2} config is empty")
+        case ("kafka", _) => load(s"$KafkaConfigPath/${args.head._2}").getOrElse(s"Khermes ${args.head._2} config is empty")
+        case ("template", _) => load(s"$TwirlTemplatePath/${args.head._2}").getOrElse(s"Khermes ${args.head._2} config is empty")
+        case ("avro", _) => load(s"$AvroConfigPath/${args.head._2}").getOrElse(s"Khermes ${args.head._2} config is empty")
+        case value => s"Arg ${value._1} is not correct."
+      }
+      a.mkString
+    }.getOrElse("Bad arguments.")
+  }
+
+  def save(args: Map[String, String], commandWord: String, config: Option[String]): Unit = {
+    Try {
```

Comment:
Take a walk on this portion of code because you can not use .get and .head. You are breaking the monad encapsulation.